sources/views/contacts.js

```diff
@@ -86,6 +64,32 @@ export default class Contacts extends JetView {
 			const firstId = webix.storage.local.get("selectedId") || contacts.getFirstId();
 			if (firstId)
 				this.contactsList.select(firstId);
-		})
+		});
+	}
+
+	removeContact(ev,id) {
+		contacts.waitSave(() => {
+			contacts.remove(id);
+			return false;
```

Comment:
I missed it. return false should be outside of promise, at the end of the method